persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/UserDAOImpl.kt

```diff
@@ -44,33 +44,43 @@ class UserDAOImpl(private val queries: UsersQueries) : UserDAO {
         )
     }
 
-    override suspend fun insertUsers(users: List<UserEntity>) {
+    override suspend fun upsertTeamMembers(users: List<UserEntity>) {
```

Comment:
Can we achieve the same result by using INSERT + ON CONFLICT E.g. https://github.com/wireapp/kalium/blob/develop/persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq#L25